README.md

```diff
@@ -224,7 +224,27 @@ Modules that can be used after you use `Plug.Router` or `Plug.Builder` to help d
   * `Plug.Debugger` - shows a helpful debugging page every time there is a failure in a request;
   * `Plug.ErrorHandler` - allows developers to customize error pages in case of crashes instead of sending a blank one;
 
+## Contributing
+
+We welcome everyone to contribute to Plug and help us tackle existing issues!
+
+Use the [issue tracker][1] for bug reports or feature requests. You may also start a discussion on the [mailing list][3] or the **[#elixir-lang][6]** channel on [Freenode][7] IRC. Open a [pull request][2] when you are ready to contribute.
+
+When submitting a pull request you should not update the `CHANGELOG.md`.
+
+If you are planning to contribute documentation, [please check our best practices for writing documentation][5].
+
+Finally, remember all interactions in our official spaces follow our [Code of Conduct][4].
+
 ## License
 
 Plug source code is released under Apache 2 License.
 Check LICENSE file for more information.
+
+  [1]: https://github.com/elixir-lang/plug/issues
+  [2]: https://github.com/elixir-lang/plug/pulls
+  [3]: https://groups.google.com/group/elixir-lang-core
+  [4]: https://github.com/elixir-lang/elixir/blob/master/CODE_OF_CONDUCT.md
+  [5]: http://elixir-lang.org/docs/stable/elixir/writing-documentation.html
+  [6]: https://webchat.freenode.net/?channels=#elixir-lang
+  [7]: http://www.freenode.net
```

Comment:
I have one suggestion: instead of using numbers for link references you can use "slugs":

```
Bla bla bla [issue tracker][plug-issues] bla bla bla


[issue tracker]: https://github.com/elixir-lang/plug/issues
```

Wdyt?